### DISCUSSION TITLE
how is the `checksum` calculated

### DISCUSSION THREAD

**Discussion by Purple-CSGO** [akiver/cs-demo-analyzer#7] [Q&A] [answered]
I've tried md5, sha1, sha256 but it does not look the same. wanna use this `checksum` for the verification purpose.

**Comment by akiver** [comment, marked as answer]
The checksum is a crc64 hash, you can find the Go code [here](https://github.com/akiver/cs-demo-analyzer/blob/main/internal/demo/demo.go#L106) and the JS equivalent [here](https://github.com/akiver/cs-demo-manager/blob/main/src/node/demo/get-demo-checksum-from-file-stats.ts#L6)
